lisa/sut_orchestrator/libvirt/ch_platform.py

```diff
@@ -189,3 +189,13 @@ def _create_node_os_disk(
                 expected_exit_code=0,
                 expected_exit_code_failure_message="Failed to copy os disk image",
             )
+
+    def _get_vmm_version(self) -> str:
+        result = ""
+        if self.host_node:
+            result = self.host_node.execute(
```

Comment:
In LISA, if the output is not huge (over 200 lines), it's better to get the output, and then filter by regexp. With this way, the LISA log contains the raw output. When the output is changed, the regexp is easier to troubleshoot than setup a new environment to get the raw output again.

Reply:
Modified to use regex.